src/utils/json_utils.cpp

```diff
@@ -59,6 +61,51 @@ std::string mp::JsonUtils::json_to_string(const QJsonObject& root) const
     return QJsonDocument(root).toJson().toStdString();
 }
 
+QJsonValue mp::JsonUtils::update_cloud_init_instance_id(const QJsonValue& cloud_init_instance_id_value,
```

Comment:
We've discussed a little elsewhere, but we need to work on this naming... Why repeat `cloud_init_instance_id` in the function name and the parameter? And any variable has a "value".

```suggestion
QJsonValue mp::JsonUtils::update_cloud_init_instance_id(const QJsonValue& id,
```

Reply:
Done.